consul/utils.go

```diff
@@ -51,3 +53,21 @@ func getServiceId(info *registry.Info) (string, error) {
 	}
 	return fmt.Sprintf("%s:%s:%d", info.ServiceName, host, port), nil
 }
+
+func GetLocalIPv4Address() (string, error) {
```

Comment:
why no use https://github.com/cloudwego/hertz/blob/develop/pkg/common/utils/network.go#L28  instead of GetLocalIPv4Address

Reply:
What about ipv6 only scenarios?